diff --git a/node_modules/vite/dist/node/chunks/dep-8f5c9290.js b/node_modules/vite/dist/node/chunks/dep-8f5c9290.js
index eeec95f..bf03107 100644
--- a/node_modules/vite/dist/node/chunks/dep-8f5c9290.js
+++ b/node_modules/vite/dist/node/chunks/dep-8f5c9290.js
@@ -19860,11 +19860,9 @@ const buildEsbuildPlugin = (config) => {
                 return null;
             }
             const target = config.build.target;
-            const minify = config.build.minify === 'esbuild' &&
-                // Do not minify ES lib output since that would remove pure annotations
-                // and break tree-shaking
-                // https://github.com/vuejs/core/issues/2860#issuecomment-926882793
-                !(config.build.lib && opts.format === 'es');
+            // Minify regardless of ESM and lib state; breaking tree-shaking
+            // is irrelevant for a Foundry package
+            const minify = config.build.minify === 'esbuild';
             if ((!target || target === 'esnext') && !minify) {
                 return null;
             }
